src/content/blog/Grain-State-of-the-Union-2020.md

```diff
@@ -56,7 +56,7 @@ The language itself isn't the only star of the show—we've put quite a lot of e
 
 ### Logo & Website Redesign
 
-![grain redesign](/blog/photos/misc/grain-redesign.png)
+![grain redesign](/src/images/misc/grain-redesign.png)
```

Comment:
This image still isn't rendering.

Reply:
That's annoying, it was working in the development mode preview